packages/next/test/fixtures/00-i18n-basepath-middleware/vercel.json

```diff
@@ -0,0 +1,101 @@
+{
+  "version": 2,
+  "builds": [{ "src": "package.json", "use": "@vercel/next" }],
+  "probes": [
+     {
+      "path": "/docs/_next/data/testing-build-id/en/dynamic/first.json?slug=first",
+      "status": 200,
+      "mustContain": "slug\":\"first\"",
+      "mustNotContain": "<html"
+    },
+    {
+      "path": "/docs/_next/data/testing-build-id/en/dynamic/123.json?slug=123",
```

Comment:
This is the specific failure case as a prerendered route will resolve correctly but a fallback generated route will not